index.rb

```diff
@@ -26,6 +26,27 @@ def show(**kwargs)
 today = Date.today
 venues = []
 
+venues << Venue.new(:name => 'Cornerstone (Berkeley)', :link => 'https://cornerstoneberkeley.com/music-venue/') do
+  URI.open(link) do |html|
+    Nokogiri(html).css('article.list-view-item').map do |article|
+      title = article.css('h1.headliners,p.supports').map(&:content).reject(&:empty?).join(', ')
+      next if title =~ /passes/i # skip links to multi-day passes
+
+      date = Date.parse(article.css('span.dates').text)
+
+      # Handle yearless dates through the december->january rollover
+      date += 365 if date.month < today.month
```

Comment:
I used `next_year` for this for brick and mortar (might be necessary for leap years?): https://github.com/davishmcclurg/shows/blob/6a13b7e51ac556d6423d1180ed3e4fa2e8649e50/index.rb#L53

Your month check is probably better than what I did though because I think there's still some weirdness around midnight.